openshift/amunInspectionWorkflow-template.yaml

```diff
@@ -0,0 +1,169 @@
+        value: "${AMUN_MEMORY}"
+      - name: registry
+        value: "${THOTH_REGISTRY}"
+      - name: thoth-infra-namespace
+        value: "${THOTH_INFRA_NAMESPACE}"
+      # Other parameters
+      - name: "target"
+        value: "inspection-result"
+
+    templates:
+    - name: main
+      dag:
+        target: "{{workflow.parameters.target}}"
+        tasks:
+        - name: inspection-build
+          arguments:
+            parameters:
+            - name: dockerfile
+              value: "{{workflow.parameters.dockerfile}}"
+            - name: specification
+              value: "{{workflow.parameters.specification}}"
+            - name: inspection-id
+              value: "{{workflow.parameters.inspection-id}}"
+            - name: allowed-failures
+              value: "{{workflow.parameters.allowed-failures}}"
+            - name: batch-name
+              value: "{{workflow.parameters.batch-name}}"
+            - name: parallelism
+              value: "{{workflow.parameters.parallelism}}"
+            - name: cpu
+              value: "{{workflow.parameters.cpu}}"
+            - name: memory
+              value: "{{workflow.parameters.memory}}"
+            - name: registry
+              value: "{{workflow.parameters.registry}}"
+            - name: thoth-infra-namespace
+              value: "{{workflow.parameters.thoth-infra-namespace}}"
+          templateRef:
+            name: inspection-build-template
+            template: main
+
+        - name: inspection-run
+          dependencies: [inspection-build]
+          arguments:
+            parameters:
+            - name: batch-name
+              value: "{{workflow.parameters.batch-name}}"
+            - name: batch-size
+              value: "{{workflow.parameters.batch-size}}"
+            - name: inspection-id
+              value: "{{workflow.parameters.inspection-id}}"
+            - name: cpu
+              value: "{{workflow.parameters.cpu}}"
+            - name: memory
+              value: "{{workflow.parameters.memory}}"
+            - name: registry
+              value: "{{workflow.parameters.registry}}"
+            - name: thoth-infra-namespace
+              value: "{{workflow.parameters.thoth-infra-namespace}}"
+          templateRef:
+            name: inspection-run-template
+            template: main
+          continueOn:
+            failed: true
+
+        - name: inspection-result
+          dependencies: [inspection-run]
+          arguments:
+            parameters:
+            - name: volume
+              value: output-volume
+            - name: namespace
+              value: "{{workflow.namespace}}"
+            - name: kind
+              value: pods
+            - name: selector
+              value: "job-name={{workflow.parameters.batch-name}}"
+            - name: registry
+              value: ${THOTH_REGISTRY}
+            - name: thoth-infra-namespace
+              value: ${THOTH_INFRA_NAMESPACE}
+          templateRef:
```

Comment:
where can we find this WorkflowTemplate?

Reply:
I will add them to the repo as discussed. Thanks for the reminder.